src/CentrifugeRouter.sol

```diff
@@ -208,6 +276,15 @@ contract CentrifugeRouter is Auth, ICentrifugeRouter {
         require(IERC20Wrapper(wrapper).withdrawTo(receiver, amount), "CentrifugeRouter/withdraw-to-failed");
     }
 
+    // --- ERC20 auth transfer ---
+    /// @inheritdoc ICentrifugeRouter
+    function authTransferFrom(address vault, address sender, address owner, address recipient, uint256 amount)
```

Comment:
Not sure if we need this here either.

Reply:
I'm also confused about this tbh. It's mentioned in the ticket directly but I dont understand why. @hieronx could you shed some light?

Reply:
Yeah my bad, `authTransferFrom` here makes no sense